docs/pages/development/installation.md

```diff
@@ -14,37 +14,32 @@ If you're just starting your project, you can create a new project from our temp
 
 <Terminal cmd={["$ npx crna -t with-dev-client"]} />
 
-If you have an existing project, you'll need to [install the package and make a few changes](installation.md) to your **AppDelegate.m**, **MainActivity.java** and **MainApplication.java**.
+Otherwise, follow this guide to add `expo-dev-client` to your existing React Native or Bare project.
 
-If you created your project with `expo init` or `create-expo-app`, or you already have `expo` and/or other Expo modules up and running, use the tabs marked **With Expo modules**.
+## 0. Set up Expo modules
 
-If you created your project with `expo init` or `create-expo-app` before SDK 43 and have `react-native-unimodules` up and running, use the tabs marked **With unimodules**. When you're ready, consider upgrading to the newer Expo modules by [following these steps](https://expo.fyi/expo-modules-migration).
+If you created your project with `npx react-native init` and do not have any other Expo packages installed, **you'll need to [install Expo modules](../bare/installing-expo-modules)** before proceeding.
 
-If you created your project with `npx react-native init` and do not have any other Expo packages installed, **starting from version `0.8.0` you'll need to [install Expo modules](../bare/installing-expo-modules)** in order to use `expo-dev-client`.
-
-> **Note**: if you are using _SDK 43 or above_, you'll need to install `expo-dev-client@0.6.0` or above.
+If you created your project with `expo init` or `create-expo-app`, or you already have `expo` and/or other Expo modules up and running, you can skip this step.
 
 ## 1. Installation
 
 Add the `expo-dev-client` library to your package.json.
 
-<InstallSection packageName="expo-development-client" cmd={["npm install expo-dev-client"]} hideBareInstructions />
-
-### iOS
+<InstallSection packageName="expo-development-client" cmd={["npx expo install expo-dev-client"]} hideBareInstructions />
 
 <Tabs tabs={["SDK 45+", "SDK below 45"]}>
 
 <Tab >
 
-Make sure that your project is configured to deploy on an iOS version _above 10_.
-To do that, open Xcode and go to General > Deployment Info, and select an iOS version of at least 11.0.
-
-<img src="/static/images/client/check_ios_version.png" style={{maxWidth: "100%" }}/>
+No additional changes are needed to install the package on SDK 45+.
```

Comment:
```suggestion
No additional changes are needed to install the package on SDK 45 and above.
```

As per the [writing style guide here](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md#symbols-as-words), let's avoid using symbols.